### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by complexsplit**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design

/kind documentation

> /kind failing-test
> /kind feature
> /kind flake

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None

```

### CODE REVIEW

**Comment by haircommander** [comment]
thanks @complexsplit 
/ok-to-test
/approve

File: install.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ To install on the following operating systems, set the environment variable $OS
 | Ubuntu 19.10     | `xUbuntu_19.10`   |
 | Ubuntu 19.04     | `xUbuntu_19.04`   |
 | Ubuntu 18.04     | `xUbuntu_18.04`   |
+| Raspberry Pi OS  | `Raspbian_10`     |
```
Comment (TomSweeneyRedHat) [inline install.md:100]: could we alphabetize this please?  move this line to 95.5.

**Review by mrunalp — APPROVED** [review]

**Pull request merged**